opentelemetry-semantic-conventions/src/opentelemetry/semconv/schemas.py

```diff
@@ -21,14 +21,14 @@ class Schemas(Enum):
     The URL of the OpenTelemetry schema version 1.23.1.
     """
 
-    V1_25_0 = "https://opentelemetry.io/schemas/v1.25.0"
```

Comment:
this file is checked with semconv code generator - https://github.com/open-telemetry/opentelemetry-python/blob/a34c284ff8e48ad6429b7f753a484fb116675071/scripts/semconv/generate.sh#L26

could you please also update the script to check for version without `v` prefix?

Reply:
I updated it and created a variable to store the tag to make it clearer. Script run without error here